rmf_fleet_adapter/test/tasks/test_Loop.cpp

```diff
@@ -178,17 +176,17 @@ SCENARIO("Test loop requests")
 
           ++completed_1_count;
         }
-        else
-          CHECK(false);
+        // else
```

Comment:
Why are we commenting out tests instead of fixing them?

Reply:
Directly remove the comments. The removal is actually a fix, as the new `ResponsiveWait`'s TaskSummmary msg will result in a false check, which in this case is untrue.